test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9454,6 +9454,16 @@ def import_hook(self, name, body):
         :returns: True on success.
         :raises: PbsManagerError
         """
+        # sync_mom_hookfiles_timeout is 15min by default
+        # Setting it to lower value to avoid the race condition at hook copy
+        srv_stat = self.status(SERVER, 'sync_mom_hookfiles_timeout')
+        try:
+            sync_val = srv_stat[0]['sync_mom_hookfiles_timeout']
+        except:
+            self.logger.info("Setting sync_mom_hookfiles_timeout to 15s")
+            self.manager(MGR_CMD_SET, SERVER,
+                         {"sync_mom_hookfiles_timeout": 15})
+
```

Comment:
I think codacy will complain at you if sync_mom_hookfiles_timeout is set.  If it is set, you set sync_val and don't use it.  Use if not 'sync_mom_hookfiles_timeout' in srv_stat[0]:

Reply:
Thanks @bhroam . I wonder why codacy did not complain on the latest commit. Do you still want me to change?

Reply:
Well you are setting a value you never plan to use.  I would change it if I was coding this, but since codacy didn't complain, it is up to you.